src/libs/ReportActionsUtils.ts

```diff
@@ -415,29 +415,23 @@ function shouldReportActionBeVisibleAsLastAction(reportAction: OnyxEntry<ReportA
  * which includes a baseURL placeholder that's replaced in the client.
  */
 function replaceBaseURL(reportAction: ReportAction): ReportAction {
-    if (!reportAction) {
-        return reportAction;
-    }
+    const policyChangeActions = new Set<string>(Object.values(CONST.REPORT.ACTIONS.TYPE.POLICYCHANGELOG));
```

Comment:
If we are going to create a set of these we should do it outside of this function so that we don't have to do it every time the function runs.

Reply:
Good point! Addressed here: fdee9b2